opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/client/ClientTracingFilter.java

```diff
@@ -61,7 +62,7 @@ public void filter(ClientRequestContext requestContext) throws IOException {
                 .asChildOf(parentSpanContext);
         }
 
-        Span span = spanBuilder.startActive(false).span();
+        Span span = spanBuilder.start();
```

Comment:
Based on the previous comments on the PR - what happens if an exception is thrown? If the scope wouldn't be closed, this also means the span is not finished?

Reply:
yes, there is an issue for it and TODO in the test